packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -614,3 +615,60 @@ export interface GetIndexerMakerStatisticsResponse {
   rewardCoefficient: BigDecimal;
   makers: IndexerMaker[];
 }
+
+export interface GetIndexerLeaderboardParams {
+  contestId: number;
+  rankType: IndexerLeaderboardRankType;
+  // Min rank inclusive
+  startCursor?: string;
+  limit?: number;
+}
+
+export interface IndexerLeaderboardParticipant {
+  subaccount: Subaccount;
+  contestId: number;
+  pnl: BigDecimal;
+  pnlRank: BigDecimal;
+  percentRoi: BigDecimal;
+  roiRank: BigDecimal;
+  // Seconds
+  updateTime: BigDecimal;
+}
+
+export interface GetIndexerLeaderboardResponse {
+  participants: IndexerLeaderboardParticipant[];
+}
+
+export interface GetIndexerLeaderboardParticipantParams {
+  contestId: number;
+  subaccount: Subaccount;
+}
+
+export interface GetIndexerLeaderboardParticipantResponse {
+  participant: IndexerLeaderboardParticipant | null;
+}
+
+export interface GetIndexerLeaderboardContestsParams {
+  contestIds: number[];
+}
+
+export interface IndexerLeaderboardContest {
+  contestId: number;
+  // NOTE: Start / End times are ignored when `period` is non-zero.
+  // Start time in seconds
+  startTime: BigDecimal;
+  // End time in seconds
+  endTime: BigDecimal;
+  // Contest duration in seconds; when set to 0, contest duration is [startTime,endTime];
+  // Otherwise, contest runs indefinitely in the interval [lastUpdated - period, lastUpdated] if active;
+  period: BigDecimal;
+  // Last updated time in Seconds
+  lastUpdated: BigDecimal;
+  totalParticipants: BigDecimal;
+  minRequiredAccountValue: BigDecimal;
```

Comment:
We should leave a comment indicating that this does not include any decimals (i.e its a float, 250 = min of 250 USDC)

Reply:
sg

Reply:
addressed